<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<!--
SPDX-FileCopyrightText: 2024 Carl Schwan <carl@carlschwan.eu>
SPDX-License-Identifier: LGPL-2.0-or-later
-->
 <class>ParagraphDropCaps</class>
 <widget class="QWidget" name="ParagraphDropCaps">
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QCheckBox" name="capsState">
       <property name="text">
        <string>Enable drop caps</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="distanceLabel">
       <property name="text">
        <string>Distance to paragraph:</string>
       </property>
       <property name="buddy">
        <cstring>distance</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KoUnitDoubleSpinBox" name="distance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="charactersLabel">
       <property name="text">
        <string>Distance to paragraph:</string>
       </property>
       <property name="buddy">
        <cstring>characters</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="characters">
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="linesLabel">
       <property name="text">
        <string>Number of lines it covers:</string>
       </property>
       <property name="buddy">
        <cstring>lines</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="lines">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
</ui>
